docker-compose/setenv.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if [ $(uname -m) == "x86_64" ] | [ $(uname -m) == "x86_64" ]; then
+    export ARCH_MEMSVC="fabric-membersrvc:x86_64-0.6.0-preview"
+    export ARCH_PEER="fabric-peer:x86_64-0.6.0-preview"
+elif [ $(uname -m) == "s390x" ]; then
+    export ARCH_MEMSVC="fabric-membersrvc:s390x-0.6.0-preview"
```

Comment:
I'm also not sure about using batch scripts here.  I'll merge this in, but I think I'd prefer to see if we can simply use an env file with variables